flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -645,6 +645,10 @@
 	@Deprecated
 	public static final String JOB_MANAGER_WEB_LOG_PATH_KEY = "jobmanager.web.log.path";
 
+	/** Config parameter indicating whether jobs can be uploaded and run from the web-frontend. */
```

Comment:
This doc isn't in sync with what the implementation actually does. It doesn't restrict anything but only logs all requests.